test/integration/backward_compatible/parallel/topic/ReliableTopicTest.js

```diff
@@ -189,4 +189,38 @@ describe('ReliableTopicTest', function () {
         const obj = clientOne.getSerializationService().toObject(item.payload);
         expect(obj).to.equal(11);
     });
+
+    it('writes and reads messages using async methods', async function () {
+        const topic = await clientOne.getReliableTopic('async');
+
+        await topic.addListener(() => {});
+
+        await topic.publish('test');
+
+        const ringbuffer = topic.getRingbuffer();
+        const seq = await ringbuffer.tailSequence();
+        const item = await ringbuffer.readOne(seq);
```

Comment:
This test logic exists with name `writes and reads messages`

Reply:
The reason is same as above.